projects/packages/waf/composer.json

```diff
@@ -12,7 +12,8 @@
 	},
 	"autoload": {
 		"files": [
-			"actions.php"
+			"actions.php",
+			"cli.php"
```

Comment:
It might be worth combining these into one file, to save a little time on startup.